cpu/log_generator.py

```diff
@@ -100,15 +101,15 @@ def test_sched_scoreboard_setup(self):
                 self.user_command, log_file)
             process.run(cmd, ignore_status=True, sudo=True, shell=True)
```

Comment:
```
 process.run(cmd, ignore_status=True, sudo=True, shell=True)
        cmd = 'echo >> "%s" 2>&1' % (log_file)
        process.run(cmd, ignore_status=True, sudo=True, shell=True)
        # Execute the command and append the output to the log file
        for i in range(self.repeat):
            cmd = 'eval "$user_command" >> "%s" 2>&1' % (log_file)
            process.run(cmd, ignore_status=True, sudo=True, shell=True)
        cmd = 'echo >> "%s" 2>&1' % (log_file)
        process.run(cmd, ignore_status=True, sudo=True, shell=True)
        for i in range(self.repeat):
            # Execute the command and append the output to the log file
            cmd = '/usr/bin/time -v $(echo %s) >> "%s" 2>&1' % (
                self.user_command, log_file)
            process.run(cmd, ignore_status=True, sudo=True, shell=True)
```

Please refactor this part to separate `test_*` methods as following:
- **test_run_command**: This method would run _user_command_ one time
- **test_run_command_with_profilers**: This method would run _user_command_ one time but with profilers enabled. (using **sysinfo** utility to run profilers for current test method only)
- **test_run_command_repeat**: This method would run _user_command_ `repeat` times

Reply:
- **test_run_with_time_repeat**: Will just be another test method to run the command through time